packages/instantsearch.js/src/components/Hits/Hits.tsx

```diff
@@ -18,8 +18,8 @@ export type HitsComponentTemplates = Required<HitsTemplates>;
 export type HitsProps = {
   results: SearchResults;
   hits: Hit[];
-  sendEvent?: SendEventForHits;
-  bindEvent?: BindEventForHits;
+  sendEvent: SendEventForHits;
+  bindEvent: BindEventForHits;
```

Comment:
ESLint was picking up on the optionally chained call to `sendEvent` (`sendEvent?.()`) because of the `no-unused-expressions` which doesn't understand optional chaining ([this one](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-expressions.md) does).

Instead of adding a a new dev dependency right now, or ignoring the rule locally, I figured making these props mandatory was better since this is an internal component and we have no reason not to pass `sendEvent` and `bindEvent`.